kotlin_request/src/main/java/com/github/paylike/kotlin_request/exceptions/apistatuscodes/ApiCodesEnum.kt

```diff
@@ -0,0 +1,32 @@
+package com.github.paylike.kotlin_request.exceptions.apistatuscodes
+
+enum class ApiCodesEnum {
```

Comment:
This seems a bit too much?

`apistatuscodes/apicodesenum` ? Think something like `api/codes.kt` would fit better? Maybe `api/statuses.kt`? Another thing is the dropping of the `Enum` part in the name that could help, as by referencing it as`ApiCodes` you should be able to see in the intellisense that it is an enum, therefore adding it to the name as well is

Reply:
Probably I didn't pay enough attention to give a proper name :D Got it!